tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestMajors.g.cs

```diff
@@ -1635,41 +1627,37 @@ public class PackageVersionsLatestMajors
 #if DEFAULT_SAMPLES
                 new object[] { string.Empty },
 #else
-#if NET48
-                new object[] { "2.2.10" },
-                new object[] { "3.8.3" },
-#endif
```

Comment:
:eyes:

Reply:
done in [0db15ca](https://github.com/DataDog/dd-trace-dotnet/pull/7070/commits/0db15ca09c2ba13fb50db81c6ca7d527592886df)